exercises/practice/word-count/.docs/hints.md

```diff
@@ -0,0 +1,32 @@
+## General
```

Comment:
@BethanyG I've looked into this, and the hints in this file should render perfectly fine.
What we need is a `## General` heading to be in the document, which you have.
My only suggestion would be to also add a top-level `# Hints` heading, which is required for the document to lint correctly. Currently there is no option to link the markdown in configlet, but we'll probably add it sometime in the future, and there is no real downside to adding it:

```suggestion
# Hints

## General
```

FYI I tested with the Elixir track and found some minor issue with their hints: https://github.com/exercism/elixir/pull/1086/files